### PR TITLE
Sara / Added language links for Google

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: gatsby-ssr.js [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
 import { WrapPagesWithLocaleContext } from './src/components/localization'
 import './src/components/localization/config'
 
 export const wrapPageElement = WrapPagesWithLocaleContext
+
+export function onRenderBody({ setHeadComponents }) {
+    const languageLinks = []
+
+    Object.keys(language_config).map(lang => {
+        languageLinks.push(<link rel="alternate" href="http://deriv.com/about" hrefLang={lang} />)
```
Comment (mustofa-binary) [inline gatsby-ssr.js]: pls search for a lifecycle to get the current page, and it should be https 🙏

Comment (sara-fs) [reply gatsby-ssr.js]: Amam it's not complete :D I had to switch to another task. :D

File: gatsby-ssr.js [hunk 2]
```diff
@@ -1,4 +1,16 @@
+import React from 'react'
+const language_config = require(`./i18n-config.js`)
 import { WrapPagesWithLocaleContext } from './src/components/localization'
 import './src/components/localization/config'
 
 export const wrapPageElement = WrapPagesWithLocaleContext
+
+// export function onRenderBody({ setHeadComponents }) {
+//     const languageLinks = []
+
```
Comment (mustofa-binary) [inline gatsby-ssr.js]: please revert this 🙏

File: src/components/containers/seo.js [hunk 3]
```diff
@@ -16,20 +17,39 @@ const SEO = ({ description, meta, title, no_index }) => {
...
+                        path
+                    }
+                }
             }
         `,
     )
-    const metaDescription = description || site.siteMetadata.description
-    const { locale: lang } = React.useContext(LocaleContext)
 
+    const metaDescription = description || queries.site.siteMetadata.description
```
Comment (mustofa-binary) [inline src/components/containers/seo.js]: @sara-fs SEO component will be rendered in every page, by doing this, we are looping on all pages in every page and pushing it in every page. you can use `path` in the query `site` instead, and push it directly

File: gatsby-ssr.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
...
```
Comment (mustofa-binary) [inline gatsby-ssr.js]: nit: can you remove this :pray:

File: src/pages/help-centre/_article.js [hunk 5]
```diff
@@ -23,6 +23,7 @@ const ContactContainer = styled.div`
 export const Article = ({ children, header }) => {
     return (
         <Layout>
+            <SEO title={localize('Help Center | Account')} />
```
Comment (mustofa-binary) [inline src/pages/help-centre/_article.js]:
```suggestion
            <SEO title={localize(`Help Center | ${header}`)} />
```

Comment (sara-fs) [reply src/pages/help-centre/_article.js]: Ah, that was test :D Sorry. Forgot to change.

File: src/components/containers/seo.js [hunk 6]
```diff
@@ -19,17 +24,40 @@ const SEO = ({ description, meta, title, no_index }) => {
...
     )
-    const metaDescription = description || site.siteMetadata.description
+
+    const metaDescription = description || queries.site.siteMetadata.description
     const { locale: lang } = React.useContext(LocaleContext)
 
+    const links = []
+    if (is_browser) {
+        var page, l
+        var currentPage = window.location.href.split('/')[3]
```
Comment (mustofa-binary) [inline src/components/containers/seo.js]: @sara-fs please use es6 🙏

**Pull request merged**